front/app/containers/Admin/tools/PowerBI/PowerBITemplates/index.tsx

```diff
@@ -42,11 +41,10 @@ const PowerBITemplates = () => {
     handleDownloadTemplate('dataflow', 'json');
   };
 
-  const hostName = API_HOST;
   const baseUrl =
-    hostName === 'localhost'
-      ? `http://localhost:${API_PORT}/api/v2/`
-      : `https://${hostName}/api/v2/`;
+    window.location.host === 'localhost'
+      ? `http://localhost:4000/api/v2/`
+      : `${window.location.origin}/api/v2/`;
```

Comment:
Don't we already have ENV variables to cover this? It's the same for any API call, right?

Reply:
That's what I was using originally `API_HOST`, but it wasn't working - looked at the codebase and not sure they're used particularly as all api calls are just absolute paths